src/signald_client/main.py

```diff
@@ -35,7 +39,7 @@ def __init__(self, signal: _Signal, mcc: MCClient, *args, **kwargs):
     def _parse_message(self, message, auto_send_receipts=True) -> Message:
         if not message.text and not message.payment:
             self.logger.warning(f"Message contained no text or payment. Not processing. {message}")
```

Comment:
Can we get rid of this logging statement now that we're raising an explicit exception with the same message?

Reply:
Actually, let's leave this here. Main reason is that I'm not explicitly checking these futures, so logging the exception is the only way you'd see an error after it's been submitted to the pool. I may fix that in the future.